include/net/net_pkt.h

```diff
@@ -241,6 +241,7 @@ struct net_pkt {
 #if defined(CONFIG_IEEE802154)
```

Comment:
In commit `net: net_pkt: Add ACK Frame Pending Bit information for ieee802154`, you can just say `net: pkt: ....` for the commit subject so no need to mention the net twice